modules/secrets-store-vault.adoc

```diff
@@ -6,11 +6,20 @@
 [id="secrets-store-vault_{context}"]
 = Mounting secrets from HashiCorp Vault
 
-You can use the {secrets-store-operator} to mount secrets from HashiCorp Vault to a CSI volume in {product-title}. To mount secrets from HashiCorp Vault, your cluster must be installed either on AWS with AWS Security Token Service (STS) or on Microsoft Azure.
+You can use the {secrets-store-operator} to mount secrets from HashiCorp Vault to a CSI volume in {product-title}.
+
+[IMPORTANT]
+====
+Mounting secrets from  HashiCorp Vault by using the {secrets-store-operator} has been tested with the following cloud providers:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+
+Other cloud providers might work, but have not been tested.
```

Comment:
Can we add "We may consider testing on other platform configurations in future."